src/main/java/racingcar/model/Winner.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winner {
+
+    private final List<String> winners;
+
+    public Winner(Cars cars) {
+        this.winners = new ArrayList<>();
+        choiceWinner(cars);
+    }
+
+    private void choiceWinner(Cars cars) {
+        int maxDistance = 0;
+
+        for (Car car : cars) {
+            int distance = car.getDistance();
+            maxDistance = compareDistance(distance, maxDistance);
+        }
+
+        for (Car car : cars) {
+            pickWinners(car, maxDistance);
+        }
+    }
+
+    private int compareDistance(int carDistance, int distance) {
+        if (carDistance >= distance) {
+            return carDistance;
+        }
+        return distance;
+    }
+
+    private void pickWinners(Car car, int distance) {
+        String carName = car.getCarName();
+        if (car.matchDistance(distance)) {
+            winners.add(carName);
+        }
+    }
+
+    public String getWinnerNames() {
+        String winnerName = String.join(", ", winners);
+        return winnerName;
```

Comment:
```suggestion
        return String.join(", ", winners);
```

1. already return을 해도 충분히 메서드 이름으로 알수 있지 않을까요?
2. ", " 이것도 상수화 해두는게 어떨까요?